docs/00.md

```diff
@@ -18,13 +18,13 @@ This is a log of me going through Cats, a functional programming library for Sca
 that is currently *experimental* and under active development.
 I'll try to follow the structure of [learning Scalaz][lsz] that I wrote in 2012 (!).
 
-Cats [website][cats] says the name is a playful shortening of the word category.
-They say coordinating programmers are like herding cats
-(as in attempting to move dozens of them in a direction like cattles, while cats have their own ideas),
-and it seems to be true at least for those of us using Scala. Keenly aware of this situation, cats list
-approachability as their first motivation.
+Cats' [website][cats] says the name is a playful shortening of the word category.
+It says coordinating programmers is like herding cats
+(as in attempting to move dozens of them in a direction like cattle, while cats have their own ideas),
+and it seems to be true at least for those of us using Scala. Keenly aware of this situation, Cats' site lists
```

Comment:
Putting `the` does make it grammatically correct. `Cats's cats` kills the fun. `the cats` maybe.